src/client/pages/Offer/Introduction/Sidebar/StartDateLabelSwitcher.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
```

Comment:
This is just a copy of the same component placed inside `Offer`. Nothing was changed.

Reply:
If nothing has changed, can we keep the one in OfferNew instead to avoid any changes that need to happen in 2 places?